### DISCUSSION TITLE
Github Profile not showing on github.com

### DISCUSSION THREAD

**Discussion by princebhatt9588** [community/community#48172] [New to GitHub]
why my profile, username or repositories are not showing on github.com when we search for them on google or on github.com ?
Can anyone please help me out!
My username is - princebhatt9588

**Comment by queenofcorgis** [comment]
Hi there! I can confirm that your profile/username is appearing for me on both GitHub and Google search.

**Reply by princebhatt9588** [reply]
oh yeah i just checked it right now but why its not showing any of my repositories there?

**Comment by melvincwng** [comment]
Hi @princebhatt9588, with regards to Google Search, you might want to refer to #44421 for more information.

In summary:
> it takes time for Google Search engine's web crawlers to crawl through websites & index them in the search engine. This process takes time, and as quoted from this [documentation](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl#:~:text=Crawling%20can%20take%20anywhere%20from,instantly%20or%20even%20at%20all.) by Google, "it can take anywhere from a few days to weeks". Just to let you know, there isn't much the community members here can do to help expedite the process of indexing your website on Google.

Hope this clarifies and answers your question 😃
